src/main/java/com/epam/ta/reportportal/core/configs/rabbit/AnalyzerRabbitMqConfiguration.java

```diff
@@ -64,7 +64,7 @@ public RabbitMqManagementClient managementTemplate(@Value("${rp.amqp.api-address
 	@Bean(name = "analyzerConnectionFactory")
 	public ConnectionFactory analyzerConnectionFactory(@Value("${rp.amqp.addresses}") URI addresses) {
 		CachingConnectionFactory factory = new CachingConnectionFactory(addresses);
-		factory.setVirtualHost(ANALYZER_KEY);
+		factory.setVirtualHost(@Value("${rp.amqp.virtualhost:” + ANALYZER_KEY + “}”);
```

Comment:
Build is failing because of incorrect string literal in expression. 
" vs ”

Reply:
Typo has been fixed. Please take a look